include/nlohmann/json.hpp

```diff
@@ -7667,7 +7671,7 @@ struct hash<nlohmann::json>
 /// @note: do not remove the space after '<',
 ///        see https://github.com/nlohmann/json/pull/679
 template<>
-struct less< ::nlohmann::detail::value_t>
+struct less<::nlohmann::detail::value_t>
```

Comment:
we should put a `// clang-format off` comment here

Reply:
We are using astyle - it should have a similar option.

Reply:
Astyle is already behaving correctly (at least I never had that change when running `make pretty`), so I believe @EvanED used clang-format.

Reply:
I am using the astyle target in the makefile.

One thing to note is that when I got a fresh checkout at the point I started, running that format produced changes -- so I wonder if I'm using a different version or something like that? I think I just apt-got the version with Ubuntu 16.04, but I'm not 100% sure.

Reply:
FWIW:

    ~$ astyle --version
    Artistic Style Version 2.05.1